<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Lab12OOPClass</class>
 <widget class="QMainWindow" name="Lab12OOPClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>683</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lab12OOP</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="ListOfStatuesLabel">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>List of Statues</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>31</y>
      <width>281</width>
      <height>371</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="ModeALayout">
     <item>
      <widget class="QListWidget" name="listOfStatues"/>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="PowerWordNameLabel">
         <property name="text">
          <string>Power word name</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEditPowerWordName"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="materialLabel">
         <property name="text">
          <string>Material</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEditMaterial"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="ageLabel">
         <property name="text">
          <string>Age</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="lineEditAge"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="corporealFormLabel">
         <property name="text">
          <string>Corporeal Form</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="lineEditCorporealForm"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="addButton">
         <property name="text">
          <string>Add</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="updateButton">
         <property name="text">
          <string>Update</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="deleteButton">
         <property name="text">
          <string>Delete</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="MyListLabel">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>10</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>My List</string>
    </property>
   </widget>
   <widget class="QPushButton" name="switch_2">
    <property name="geometry">
     <rect>
      <x>314</x>
      <y>140</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>388</x>
      <y>32</y>
      <width>281</width>
      <height>371</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QListWidget" name="Mylist"/>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="savePowerWordNameLabelModeB">
         <property name="text">
          <string>Power word name</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEditPowerWordNameModeB"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="MaterialLabelModeB">
         <property name="text">
          <string>Material</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEditMaterialModeB"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="LabelAgeModeB">
         <property name="text">
          <string>Age</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="lineEditAgeModeB"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="LabelCorporealFormModeB">
         <property name="text">
          <string>Corporeal Form</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="lineEditCorporealFormModeB"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout_3">
       <item row="1" column="0">
        <widget class="QPushButton" name="OpenButton">
         <property name="text">
          <string>Open</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="NextButton">
           <property name="text">
            <string>Next</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="SaveButton">
           <property name="text">
            <string>Save</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="FilterButton">
           <property name="text">
            <string>Filter</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>683</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Lab12OOP.qrc"/>
 </resources>
 <connections/>
</ui>
